### PR TITLE
output actual AirbyteMessages for cdc

### PR DESCRIPTION
**Pull request by jrhizor**
The main things I wanted address:
1. setting `_ab_cdc_deleted_at` to null instead of omitting it
2. the fact that inserts and updates look identical since they just set updated_at

### CODE REVIEW

File: airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java [hunk 1]
```diff
@@ -162,13 +156,13 @@ public JsonNode toJdbcConfig(JsonNode config) {
 
       // todo - this obviously needs to actually do something.
       final Predicate<Object> hasReachedLsnPredicate = (r) -> false;
-      final Iterator<JsonNode> queueIterator = Queues.toStream(queue).iterator();
-      final AbstractIterator<JsonNode> iterator = new AbstractIterator<>() {
+      final Iterator<AirbyteMessage> queueIterator = Queues.toStream(queue).iterator();
+      final AbstractIterator<AirbyteMessage> iterator = new AbstractIterator<>() {
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java:160]: this in-line class definition really threw me off while reading this code. Why don't we refactor it out of this method? The method is pretty large right now which makes it hard to follow.

**Review by sherifnada — CHANGES_REQUESTED** [review]
Requesting changes because the method for getting CDC iterators is hard to follow. Also, why is setting deleted_at to null better than just omitting it? it should end up being the same thing in the destination, right?

File: airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java (hunk 1 above)
Comment (cgardens) [reply airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java:160]: there's still a lot we're trying to lock down here right now. i agree, we will want to refactor this to make the responsibility of each piece more clear and testable. can we get a pass on this for now while we are still making pretty big code changes in this area?

Comment (sherifnada) [reply airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java:160]: 👍🏼

**Review by sherifnada — APPROVED** [review]
